packages/tab-list/src/tab-list.ts

```diff
@@ -28,6 +29,14 @@ const availableArrowsByDirection = {
     horizontal: ['ArrowLeft', 'ArrowRight'],
 };
 
+declare global {
+    interface Document {
+        fonts?: {
+            ready: Promise<void>,
```

Comment:
For uniformity, can you move the following from the `dis/connecedCallback`s up here as well?
```
removeEventListener: (
    name: string,
    callback: () => void
) => void;
```